### PR TITLE
Gen8: Tiny Libero fix

### PR DESCRIPTION
**Pull request by ErwanBeurier**
Just adding missing parentheses in an if.

### CODE REVIEW

File: Data/Scripts/012_Battle/008_BattleHandlers_Items.rb [hunk 1]
```diff
@@ -382,8 +382,8 @@
 
 BattleHandlers::PriorityBracketChangeItem.add(:CUSTAPBERRY,
   proc { |item,battler,subPri,battle|
-    next if !battler.pbCanConsumeBerry?(item)
-    next 1 if subPri<1 && (!battler.effects[PBEffects::PriorityAbility] && b.hasActiveAbility?(:QUICKDRAW))
```
Comment (Maruno17) [inline Data/Scripts/012_Battle/008_BattleHandlers_Items.rb:386]: This looks like something that shouldn't have been deleted.

File: Data/Scripts/012_Battle/001_Battler/006_Battler_AbilityAndItem.rb [hunk 2]
```diff
@@ -126,8 +126,13 @@ def pbOnAbilityChanged(oldAbil)
   #=============================================================================
   # Held item consuming/removing
   #=============================================================================
-  def pbCanConsumeBerry?(_item,alwaysCheckGluttony=true)
-    return false if isUnnerved?
+  def canConsumeBerry?
+    return false if @battle.pbCheckOpposingAbility(:UNNERVE, @index)
+    return true
+  end
```
Comment (Maruno17) [inline Data/Scripts/012_Battle/001_Battler/006_Battler_AbilityAndItem.rb:132]: master now does its own thing with Unnerve to condense its mentions into one place. This probably makes def isUnnerved? redundant, but I believe said method lists some Gen 8 abilities that will need to be moved into here. Make sure def isUnnerved? is no longer used and deleted, and make sure def canConsumeBerry? now does all the things it needs to.

**Review by Maruno17 — CHANGES_REQUESTED** [review]
I spotted two things that need addressing.

**Comment by ErwanBeurier** [comment]
The changes you spotted are in the master branch. They appear in this PR because I pulled the master branch to get the Copycat bugfix.
I reverted this pull, so it should be good now.

**Pull request merged**